static/components/AppAround.js

```diff
@@ -108,12 +109,11 @@
 	        });
 	      });
 	      navigator.geolocation.getCurrentPosition(function (position) {
-	        var userPosition = [position.coords.longitude, position.coords.latitude];
-	        moveTo(userPosition);
-	        currentMarker(userPosition);
-	        console.log(userPosition);
+	        localStorage.setItem('userLat', position.coords.latitude);
```

Comment:
I would denitely store the userLatLng in one _localStorage_ variable. They are tightly coupled and will never work apart.